service/controller/v1/resource_set.go

```diff
@@ -144,24 +141,11 @@ func NewResourceSet(config ResourceSetConfig) (*controller.ResourceSet, error) {
 		return ctx, nil
 	}
 
-	handlesFunc := func(obj interface{}) bool {
-		customResource, err := key.ToCustomResource(obj)
-		if err != nil {
-			return false
-		}
-
-		if key.VersionBundleVersion(customResource) == VersionBundle().Version {
-			return true
-		}
-
-		return false
-	}
-
```

Comment:
We are moving from version bundle to label selector. Dropping this for now and will add label selector in later PR.

Reply:
How does the deployment work though? I feel like dropping this here should be the last step to activate the new behaviour.

Reply:
This operator is not deployed yet.